### PR TITLE
add helpText guidelines

### PR DESCRIPTION
**Pull request by kaycebasques**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -310,6 +310,25 @@ If you are simply adding additional audits/gatherers or tweaking flags, you can
 }
 ```
 
+### helpText guidelines
```
Comment (ebidel) [inline readme.md]: would this be better in https://github.com/GoogleChrome/lighthouse/blob/master/CONTRIBUTING.md?

File: readme.md [hunk 2]
```diff
@@ -310,6 +310,25 @@ If you are simply adding additional audits/gatherers or tweaking flags, you can
...
+
+Keep the `helpText` of an audit as short as possible. When a reference doc for the audit exists on
+developers.google.com/web, the `helpText` should only explain *why* the user should care
+about the audit, not *how* to fix it.
+
+Do:
+
+    Serve images that are smaller than the user's viewport to save cellular data and
+    improve load time.
```
Comment (patrickhulce) [inline readme.md]: Maybe add a markdown link in the "Do" example so it's clear you should do this when you have a link :)

File: readme.md (hunk 1 above)
Comment (kaycebasques) [inline readme.md]: Missed that. Ya, it's better there probably

Comment (kaycebasques) [inline readme.md]: done

File: readme.md (hunk 2 above)
Comment (kaycebasques) [inline readme.md]: done

**Pull request merged**